commit fcafcd162c843364dc2bb8d57bd239c41cfd122c
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Mon Sep 25 01:55:02 2017 +0200

    hurd: Fix exposition of s/gettimeofday through timespec_s/get
    
    conform/ISO11/time.h/linknamespace complains that using timespec_get exposes
    gettimeofday.
    
    conform/POSIX/time.h/linknamespace complains that using clock_settime
    exposes settimeofday.
    
            * sysdeps/unix/clock_gettime.c (realtime_gettime, __clock_gettime): Use
            __gettimeofday instead of gettimeofday.
            * sysdeps/unix/clock_settime.c (__clock_settime): Use __settimeofday
            instead of settimeofday.

diff --git a/sysdeps/unix/clock_gettime.c b/sysdeps/unix/clock_gettime.c
index 5262066f51..c4cf49367e 100644
--- a/sysdeps/unix/clock_gettime.c
+++ b/sysdeps/unix/clock_gettime.c
@@ -79,7 +79,7 @@ static inline int
 realtime_gettime (struct timespec *tp)
 {
   struct timeval tv;
-  int retval = gettimeofday (&tv, NULL);
+  int retval = __gettimeofday (&tv, NULL);
   if (retval == 0)
     /* Convert into `timespec'.  */
     TIMEVAL_TO_TIMESPEC (&tv, tp);
@@ -103,7 +103,7 @@ __clock_gettime (clockid_t clock_id, struct timespec *tp)
     case CLOCK_REALTIME:
       {
 	struct timeval tv;
-	retval = gettimeofday (&tv, NULL);
+	retval = __gettimeofday (&tv, NULL);
 	if (retval == 0)
 	  TIMEVAL_TO_TIMESPEC (&tv, tp);
       }
diff --git a/sysdeps/unix/clock_settime.c b/sysdeps/unix/clock_settime.c
index e744cae6a9..067e279e2a 100644
--- a/sysdeps/unix/clock_settime.c
+++ b/sysdeps/unix/clock_settime.c
@@ -89,7 +89,7 @@ __clock_settime (clockid_t clock_id, const struct timespec *tp)
 	struct timeval tv;						      \
 	TIMESPEC_TO_TIMEVAL (&tv, tp);					      \
 									      \
-	retval = settimeofday (&tv, NULL);				      \
+	retval = __settimeofday (&tv, NULL);				      \
       } while (0)
 
 #ifdef SYSDEP_SETTIME
